relayer/relayer.go

```diff
@@ -0,0 +1,170 @@
+package relayer
+
+import (
+	"fmt"
+	"math/big"
+	"net"
+	"path"
+	"strconv"
+	"strings"
+
+	"github.com/0xPolygon/polygon-edge/contracts"
+	"github.com/0xPolygon/polygon-edge/tracker"
+	"github.com/0xPolygon/polygon-edge/txrelayer"
+	"github.com/0xPolygon/polygon-edge/types"
+
+	hcf "github.com/hashicorp/go-hclog"
+	"github.com/umbracle/ethgo"
+	"github.com/umbracle/ethgo/abi"
+	"github.com/umbracle/ethgo/jsonrpc"
+)
+
+var commitEvent = abi.MustNewEvent(`event NewBundleCommit(uint256 startId, uint256 endId, bytes32 root)`)
+
+type Relayer struct {
```

Comment:
Would `StateSyncExecutor` be semantically more adequate name?

Reply:
We have landed to `StateSyncRelayer` name eventually 🙂

Reply:
https://github.com/0xPolygon/polygon-edge/pull/1044#issuecomment-1363895539